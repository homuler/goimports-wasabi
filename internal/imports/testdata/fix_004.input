// insert a single import declaration
/*comment*/package foo/*comment*/;func HelloWorld() { fmt.Println("Hello, World!") }
