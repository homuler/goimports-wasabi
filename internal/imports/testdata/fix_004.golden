// insert a single import declaration
/*comment*/ package foo /*comment*/
import "fmt"

func HelloWorld() { fmt.Println("Hello, World!") }
